Products/ZenUI3/browser/resources/js/zenoss/zenoss.js

```diff
@@ -1315,6 +1315,24 @@ Zenoss.util.ipv6wrap = function(string) {
     return string;
 };
 
+/**
+ * Helper fn to get Ext component by 'itemId' starting from from some target component.
+ * We move up in Ext cmp hierarchy and search ('down') for cmp by it's 'itemId'.
+ * @param itemId
+ * @param targetCmp
+ * @returns {*}
+ */
+Zenoss.getCmp = function(itemId, targetCmp) {
+    var cmp = null, target;
+    if (itemId) {
+        while (!cmp && targetCmp) {
+            target = targetCmp.down ? targetCmp : targetCmp.getEl();
```

Comment:
if _targetCmp_ has no "down" fn get it's el and and use "down" of element.
We can't use element.down fn all the time because it's not always searching for components inside itself.